app/uk/gov/hmrc/thirdpartyapplication/models/db/ApplicationData.scala

```diff
@@ -41,7 +41,8 @@ case class ApplicationData(id: UUID,
                            rateLimitTier: Option[RateLimitTier] = Some(BRONZE),
                            environment: String = Environment.PRODUCTION.toString,
                            checkInformation: Option[CheckInformation] = None,
-                           blocked: Boolean = false) {
+                           blocked: Boolean = false,
+                           cidrBlocks: Set[String] = Set.empty) {
```

Comment:
Ass comment on `app.routes`, this should probably be called `ipWhitelist` (or similar)